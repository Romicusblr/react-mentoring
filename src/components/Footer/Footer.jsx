import React from 'react';
import classNames from 'classnames';
import PropTypes from 'prop-types';
import Logo from '../Logo';
import style from './Footer.module.css';

const Footer = ({ className, ...other }) => (
  <footer
    className={classNames(style.footer, className)}
    {...other}
  >
    <Logo />
  </footer>
);

Footer.propTypes = {
  className: PropTypes.string,
};

Footer.defaultProps = {
  className: '',
};

export default Footer;
