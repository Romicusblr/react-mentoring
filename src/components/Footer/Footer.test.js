import Footer from '.';

fdescribe('Footer', () => {
  it('should be selectable by class "footer"', () => {
    expect(shallow(<Footer />).is('.footer')).toBe(true);
  });

  it('should match to snapshot', () => {
    const snap = shallow(<Footer />);
    expect(snap).toMatchSnapshot();
  });
});
