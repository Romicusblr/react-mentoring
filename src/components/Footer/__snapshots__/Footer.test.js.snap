// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match to snapshot 1`] = `
<footer
  className="footer"
>
  <Logo
    className=""
  />
</footer>
`;
